File: testing/driver/src/main/java/net/corda/testing/driver/node/Member.java
package net.corda.testing.driver.node;

import net.corda.v5.base.annotations.DoNotImplement;
import net.corda.v5.base.types.MemberX500Name;
import org.jetbrains.annotations.NotNull;

@DoNotImplement
public interface Member {
    @NotNull
    MemberX500Name getName();

    @NotNull
    MemberStatus getStatus();

    void setStatus(@NotNull MemberStatus status);
}

File: testing/cpbs/atomic-swap-contracts/src/main/kotlin/com/r3/corda/atomic/swap/states/Member.kt
package com.r3.corda.atomic.swap.states

import net.corda.v5.base.annotations.CordaSerializable
import net.corda.v5.base.types.MemberX500Name
import java.security.PublicKey

@CordaSerializable
class Member(
    val name: MemberX500Name,
    val ledgerKey: PublicKey
) {
    override fun toString(): String {
        return "Member{" + "name=" + name + '}'
    }
}